### DISCUSSION TITLE
Load original image only on Wifi

### DISCUSSION THREAD

**Discussion by bartalta** [immich-app/immich#5617] [Feature Request]
I'd like to be able to toggle the load original image in the image viewer quality section, but only on Wifi.

**Comment by andresanches** [comment]
+1, but for videos as well

**Comment by taymcmillen** [comment]
I was coming here to request something similar.... In asset viewer, I can choose Force Original Video.

It would be great to be able to choose a setting for, Force Original Video while on Local Network
